diff --git a/index.js b/index.js
index f26482978d544bf39828bd152f2369d53413bc27..f071082d4dd1fb42a9e1497d0ca64363e75e2f4d 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
 /*! chokidar - MIT License (c) 2012 Paul Miller (paulmillr.com) */
 import { EventEmitter } from 'node:events';
-import { stat as statcb, Stats } from 'node:fs';
+import { stat as statcb } from 'node:fs';
 import { readdir, stat } from 'node:fs/promises';
 import * as sp from 'node:path';
 import { readdirp, ReaddirpStream } from 'readdirp';
